fab/inventory/staging

```diff
@@ -36,7 +36,7 @@ hqdb0-staging.internal.commcarehq.org
 hqdb0-staging.internal.commcarehq.org
 
 [elasticsearch]
-hqdjango2-staging.internal.commcarehq.org elasticsearch_node_name=hqdjango2-staging
+hqes1-staging.internal.commcarehq.org
```

Comment:
I think this might need `elasticsearch_node_name=hqes1-staging` at the end of it.  It looks like that IS the current node name, but I think that might be because I deployed a different version of this file earlier.